modules/applications/pages/ui-builder/download-and-launch.adoc

```diff
@@ -1,18 +1,34 @@
 = Download and launch Bonita UI Builder
 :page-aliases: applications:download-and-launch.adoc
-:description: Describes how to download and launch Bonita UI Builder.
+:description: Lean how to setup Bonita UI Builder environment ready for development.
```

Comment:
```suggestion
:description: Learn how to download and launch Bonita UI Builder.
```

Reply:
@marcobonita why do the changes I propose not suit you?

Reply:
@educhastenier ce qui me gène c'est d'ajouter un 3ème pré-requis et d'afficher un gros bloc warning d'entrée de jeu. Je propose une approche plus "douce" ici : https://github.com/marcobonita/bonita-doc/blob/patch-44/modules/applications/pages/ui-builder/download-and-launch.adoc